builtin/logical/postgresql/backend.go

```diff
@@ -64,8 +64,8 @@ type backend struct {
 
 // DB returns the database connection.
 func (b *backend) DB(ctx context.Context, s logical.Storage) (*sql.DB, error) {
-	b.logger.Trace("postgres/db: enter")
-	defer b.logger.Trace("postgres/db: exit")
+	b.logger.Debug("postgres/db: enter")
```

Comment:
Did we missing removing the prefixes from the postgres files?

Reply:
Looks like it yes